Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentController.cpp

```diff
@@ -89,10 +157,11 @@ namespace MiniAudio
         {
             ma_engine_listener_set_position(engine, m_config.m_listenerIndex, world.GetTranslation().GetX(), world.GetTranslation().GetY(), world.GetTranslation().GetZ());
 
-            const AZ::Vector3 forward = world.TransformVector(AZ::Vector3::CreateAxisY(-1.f));
+            const AZ::Vector3 forward = world.TransformVector(AZ::Vector3::CreateAxisY(1.f));
```

Comment:
Minor, you could use world.BasisY() and world.BasisZ() to avoid a little bit of math.

Reply:
See 87d1400dfa20a3850ea7f1a1a4712de7730b4318